big_tests/tests/inbox_helper.erl

```
@@ -258,15 +259,19 @@ make_inbox_form(GetParams) ->
     End -> [escalus_stanza:field_el(<<"end">>, <<"text-single">>, End)]
   end,
   FormTypeL = [escalus_stanza:field_el(<<"FORM_TYPE">>, <<"hidden">>, ?NS_ESL_INBOX)],
-  Fields = FormTypeL ++ OrderL ++ StartL ++ EndL,
+  HiddenReadL = [escalus_stanza:field_el(<<"hidden_read">>, <<"text-single">>,  
+                                         bool_to_bin(maps:get(hidden_read, GetParams, false)))],
+  Fields = FormTypeL ++ OrderL ++ StartL ++ EndL ++ HiddenReadL,
   escalus_stanza:x_data_form(<<"submit">>, Fields).
 
-order_to_bin(asc) -> <<"asc">>;
-order_to_bin(desc) -> <<"desc">>.
-
 %% ---------------------------------------------------------
 %% 1-1 helpers
 %% ---------------------------------------------------------
+order_to_bin(asc) -> <<"asc">>;
+order_to_bin(desc) -> <<"desc">>.
+
+bool_to_bin(true) -> <<"true">>;
+bool_to_bin(_) -> <<"false">>.
```

```
bool_to_bin(true) -> <<"true">>;
bool_to_bin(false) -> <<"false">>.
```
otherwise let it crash